indico/modules/events/sessions/templates/display/session_list.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {% trans %}My Sessions{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <table id="sessions-list" class="i-table-widget">
+        <thead>
+            <tr>
+                <th>{% trans %}Name{% endtrans %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for sess in sessions %}
+                <tr>
+                    <td>
+                        {% if session.user and sess.can_manage(session.user) %}
```

Comment:
No need to check for `session.user`. It'll never be None since that page requires you to be logged in (and `can_manage` handles a None user just fine).

Also, I think we'll also allow coordinators to access the session list since they can manage some parts of their session